packages/rest-crud/README.md

```diff
@@ -29,18 +37,66 @@ npm install --save @loopback/rest-crud
    >(Product, {basePath: '/products'});
    ```
 
-4. Set up dependency injection for the ProductController.
+2. Set up dependency injection for the ProductController.
 
    ```ts
    inject('repositories.ProductRepository')(ProductController, undefined, 0);
    ```
 
-5. Register the controller with your application.
+3. Register the controller with your application.
 
    ```ts
    app.controller(ProductController);
    ```
 
+### Creating a CRUD repository
+
+Use the `defineCrudRepositoryClass` method to create named repositories (based
+on the Model) for your app.
+
+Usage example:
+
+```ts
+const db = new juggler.DataSource({connector: 'memory'});
+const ProductRepository = defineCrudRepositoryClass(Product);
+const repo = new ProductRepository(db);
```

Comment:
I find it confusing that the Controller section is showing DI + app.controller, but the Repository section is showing manual approach, which also does not take into account different lifetimes of a datasource (one instance for entire app life) vs. a repository (new repository created for each request).